zetta_utils/db_annotations/layer_group.py

```diff
@@ -0,0 +1,64 @@
+"""Manage layer groups in a DB Layer."""
+
+from __future__ import annotations
+
+import uuid
+
+from zetta_utils.layer.db_layer import DBRowDataT, build_db_layer
+from zetta_utils.layer.db_layer.datastore import DatastoreBackend
+
+from . import constants
+
+DB_NAME = "layer_groups"
+INDEXED_COLS = ("name", "layers", "created_by", "modified_by")
```

Comment:
Would it make more sense to keep a layer group per collection? So that the name for the same layers can be different depending on collection